src/database-fs.ts

```diff
@@ -35,11 +37,10 @@ export async function * iterKeys (): AsyncGenerator<string> {
   }
 }
 
+// deno-lint-ignore require-await
 export async function readData (key: string): Promise<Uint8Array | string | void> {
-  // Necessary here to thwart path traversal attacks.
   checkKey(key)
-  return await Deno.readFile(path.join(dataFolder, key))
-    .catch(_err => undefined)
+  return Deno.readFile(path.join(dataFolder, key)).catch(() => undefined)
```

Comment:
This would not need an `// deno-lint-ignore require-await` comment if the function were not marked `async`.

So the `async` label can simply be removed, and then you should be able to remove the comment too